aws/rust-runtime/aws-sigv4/src/lib.rs

```diff
@@ -29,14 +30,11 @@ pub mod event_stream;
 pub mod http_request;
 
 /// Parameters to use when signing.
+// #[derive(Debug)] is safe because `Identity` handles redaction.
```

Comment:
`Identity` doesn't handle redaction, but the data inside of it is _supposed to_.

Reply:
I'll clarify that.